db/migrate/20160430092324_add_default_false_value_to_notifications.rb

```diff
@@ -1,5 +1,8 @@
 class AddDefaultFalseValueToNotifications < ActiveRecord::Migration
```

Comment:
This doesn't need to be an up/down migration; the `change` method using a `change_column` call is able to handle forwards and backwards migrations.